docs/kb/semgrep-code/reduce-false-positives.md

```diff
@@ -0,0 +1,88 @@
+---
+slug: reduce-false-positives
+title: Reduce false positives
+hide_title: true
+description: Learn different strategies to reduce false positives in your Semgrep OSS scans.
+tags:
+  - Semgrep Code
+  - Semgrep OSS
+---
+
+# Reduce false positives in `semgrep scan`
+
+The `semgrep scan` command can be used to quickly perform SAST scans. However, you may encounter false positives as you work through your findings. This document presents different strategies to reduce false positives and increase true positives in your scans.
+
+## Customize your rules
```

Comment:
This is actually the only way I am aware of for people to improve the false positive rate on OSS - either:
- write a better rule
- take it out

And I decided to include it because it's still a valid solution if the user absolutely doesn't want to sign in to Semgrep.

HOWEVER, I am wondering, should it be at the top of the doc or further down?

Reply:
Do we also want to add a section or a new page for supply-chain false positives?

Will share some more context over slack.

Reply:
@zzeleznick How's this:
![image](https://github.com/user-attachments/assets/452b167d-a995-4b48-a1ef-26306a595077)